item-impl/src/main/java/com/example/auction/item/impl/CompletionStageUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.example.auction.item.impl;
+
+import akka.Done;
+import com.datastax.driver.core.BoundStatement;
+import com.lightbend.lagom.javadsl.persistence.cassandra.CassandraReadSide;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+final class CompletionStageUtils {
+    private CompletionStageUtils() {
+        throw new Error("no instances");
+    }
+
+    static <R, X extends RuntimeException> Function<Optional<R>, R> throwIfEmpty(Supplier<X> exceptionThrower) {
+        return result -> result.orElseThrow(exceptionThrower);
+    }
+
+    static CompletionStage<List<BoundStatement>> completedStatements(BoundStatement... statements) {
+        return CassandraReadSide.completedStatements(Arrays.asList(statements));
+    }
```

Comment:
This one in particular seems like a pretty natural convenience to add to the ones that are already there.